configCivcraft.yml

```diff
@@ -3260,11 +3257,11 @@ recipes:
     input:
       iron:
         material: IRON_INGOT
-        amount: 8
+        amount: 10
```

Comment:
Those rates are in line with those for all the other tiers. Why changes this specifically?